hw/ip/aes/doc/checklist.md

```diff
@@ -104,21 +104,21 @@ Review        | Signoff date            | Not Started |
  Type         | Item                                  | Resolution  | Note/Collaterals
 --------------|---------------------------------------|-------------|------------------
 Documentation | [DV_PLAN_DRAFT_COMPLETED][]           | Done        |
-Documentation | [TESTPLAN_COMPLETED][]                | Not Started |
-Testbench     | [TB_TOP_CREATED][]                    | Not Started |
-Testbench     | [PRELIMINARY_ASSERTION_CHECKS_ADDED][]| Not Started |
-Testbench     | [TB_ENV_CREATED][]                    | Not Started |
-Testbench     | [RAL_MODEL_GEN_AUTOMATED][]           | Not Started |
-Testbench     | [TB_GEN_AUTOMATED][]                  | Not Started |
-Tests         | [SANITY_TEST_PASSING][]               | Not Started |
-Tests         | [CSR_MEM_TEST_SUITE_PASSING][]        | Not Started |
-Tool Setup    | [ALT_TOOL_SETUP][]                    | Not Started |
-Regression    | [SANITY_REGRESSION_SETUP][]           | Not Started |
-Regression    | [NIGHTLY_REGRESSION_SETUP][]          | Not Started |
+Documentation | [TESTPLAN_COMPLETED][]                | Not Started | Not sure if the testplan is complete yet.
```

Comment:
not completed but definitely started

Reply:
For V1 qualification, testplan and DV plan needs to be reviewed by designers and peers. I dont think we have had that done yet.

Reply:
https://docs.opentitan.org/doc/project/hw_stages/#hardware-verification-stages